src/transformState.js

```diff
@@ -5,7 +5,19 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    if (action.type === 'addProperties') {
```

Comment:
Better use `switch-case` construction. Or use just `if`, cause there are only unique cases here. It will be more clearer. It's a mentor-preferred solution 😬

Reply:
code looks much cleaner with switch statement, thanks!